src/dstack/_internal/core/models/profiles.py

```diff
@@ -167,6 +168,24 @@ def validate_time_window(cls, v: Union[int, str]) -> int:
         return v
 
 
+class Schedule(CoreModel):
+    cron: Annotated[
```

Comment:
I thought we want to support multiple cron expression.

Reply:
yeah, I can do it next week since it'll be some more work.

Reply:
Supported multiple crons